groundgrid:
  ros__parameters:
    point_count_cell_variance_threshold: 10
    max_ring: 1024
    groundpatch_detection_minimum_threshold: 0.01
    distance_factor: 0.0001
    minimum_distance_factor: 0.0005
    miminum_point_height_threshold: 0.3
    minimum_point_height_obstacle_threshold: 0.1
    outlier_tolerance: 0.1
    ground_patch_detection_minimum_point_count_threshold: 0.25
    patch_size_change_distance: 20
    occupied_cells_decrease_factor: 5.0
    occupied_cells_point_count_factor: 20
    min_outlier_detection_ground_confidence: 1.25
    hole_negative_residual_threshold: 0.2
    hole_cost_sigma: 0.1
    hole_persistence_increase: 0.25
    hole_persistence_decay: 0.1
    hole_no_observation_decay: 0.02
    interpolation_height_sigma: 0.2
    interpolation_slope_sigma: 0.35
    interpolation_anisotropic_direction_factor: 1.5
    interpolation_min_confidence: 0.01
    thread_count: 8
